Source/MonoGame.Extended/Maps/Tiled/TiledMap.cs

```diff
@@ -20,19 +25,21 @@ public TiledMap(GraphicsDevice graphicsDevice, int width, int height, int tileWi
             _graphicsDevice = graphicsDevice;
             _layers = new List<TiledLayer>();
             _tilesets = new List<TiledTileset>();
+            Orientation = orientation;
         }
         
         private readonly List<TiledTileset> _tilesets;
         private readonly GraphicsDevice _graphicsDevice;
         private readonly List<TiledLayer> _layers;
         
-        public int Width { get; private set; }
-        public int Height { get; private set; }
-        public int TileWidth { get; private set; }
-        public int TileHeight { get; private set; }
+        public int Width { get; }
+        public int Height { get; }
+        public int TileWidth { get; }
+        public int TileHeight { get; }
```

Comment:
Yeah, I'm not sure if it's worth it. I've noticed at least a few contributors have been using it and I'd like to be using it myself. Our build machine still has VS2013 but that'd be an easy upgrade. I wonder if compiling on Linux or with Xamarin Studio has any issues.